src/presageworker.cpp

```diff
@@ -63,7 +63,8 @@ void PresageWorker::predict()
         return;
 
     QString lang = m_language;
-    if (m_predictor->contextStream(m_last_id, lang, *m_buffer)) {
+    if (m_predictor->contextStream(m_last_id, lang, *m_buffer, m_forcePrediction)) {
```

Comment:
What if we would add a boolean parameter to the with default false value: PresageWorker::predict(bool force = false)
This way we could spare a member variable. It is good and working this way too, just somehow I felt the argument based approach more "natural" and self-descripting. What do you think?